includes/ucf-social-common.php

```diff
@@ -124,7 +123,90 @@ function ucf_social_icons_display_after( $atts ) {
 		}
 
 		/**
-		* Enqueue css for the social icons
+		* Displays the social social links
+		* @author RJ Bruneel
+		* @since 1.0
+		* @param array $atts | Assoc. array of shortcode options
+		* @return string
+		**/
+		public static function display_social_links( $atts ) {
+			$before = self::ucf_social_links_display_before( $atts );
+			if ( has_filter( 'ucf_social_display_before' ) ) {
+				$before = apply_filters( 'ucf_social_display_before', $before, $atts );
+			}
+
+			$content = self::ucf_social_links_display( $atts );
+			if ( has_filter( 'ucf_social_display' ) ) {
+				$content = apply_filters( 'ucf_social_display', $content, $atts );
+			}
+
+			$after = self::ucf_social_links_display_after( $atts );
+			if ( has_filter( 'ucf_social_display_after' ) ) {
+				$after = apply_filters( 'ucf_social_display_after', $after, $atts );
+			}
+
+			return $before . $content . $after;
+		}
+
+		/**
+		* Display the content before the social links
+		* @author RJ Bruneel
+		* @since 1.0
+		* @return string
+		**/
+		function ucf_social_links_display_before( $atts ) {
+			ob_start();
+		?>
+			<aside class="ucf-social-links">
+		<?php
+			echo ob_get_clean();
+		}
+
+		/**
+		* Display the social links content
+		* @author RJ Bruneel
+		* @since 1.0
+		* @param array $atts | Assoc. array of shortcode options
+		* @return string
+		**/
+		function ucf_social_links_display( $atts ) {
+			$atts = shortcode_atts( array(
+				'size'  => 'md'
+			), $atts );
+
+			$permalink = urlencode( get_permalink( $_POST->ID ) );
```

Comment:
Referencing `global $post` when getting the permalink is probably a better practice here (I think $_POST is actually intended to be used for referencing form data).
e.g.
```
global $post;
if ( !$post ) { return; }  // back out if there's no post data to reference
$permalink = urlencode( get_permalink( $post->ID ) );
```